sopel/coretasks.py

```diff
@@ -59,6 +59,12 @@ def auth_after_register(bot):
             account + ' ' + password
         ))
 
+    elif bot.config.core.auth_method == 'userserv':
+        account = bot.config.core.auth_username
+        password = bot.config.core.auth_password
+        bot.msg("USERSERV", "LOGIN {account} {password}".format(
```

Comment:
The UserServ name shouldn't be hardcoded. See #1238, which uses `bot.config.core.auth_target` if configured.